syntax = "proto3";

package sbai_geographic_protos;

import "sbai_geographic_protos/geo_pose.proto";
import "sbai_std_protos/header.proto";

message GeoPoseStamped {
    sbai_std_protos.Header header = 1;
    sbai_geographic_protos.GeoPose pose = 2;
}
